operations/helm/charts/mimir-distributed/values.yaml

```diff
@@ -4088,6 +4092,17 @@ tokengenJob:
   # -- The name of the PriorityClass for tokenjobgen pods
   priorityClassName: null
 
+# -- kubetclImage is used in the enterprise provisioner and tokengen jobs
+kubectlImage:
+  # -- The Docker registry
+  registry: docker.io
+  # -- Docker image repository
+  repository: bitnami/kubectl
+  # -- Overrides the image tag whose default is the chart's appVersion
+  tag: null
```

Comment:
should this be `latest`? Also the comment looks outdated

Reply:
fixed